velox/functions/sparksql/aggregates/tests/CMakeLists.txt

```diff
@@ -34,6 +35,7 @@ target_link_libraries(
   velox_exec_test_lib
   velox_vector_test_lib
   velox_functions_aggregates_test_lib
+  velox_functions_spark
```

Comment:
Wondering why we do we need this dependency now?

Reply:
We need to use `array_sort` function in new added tests to ensure results are comparable.